### DISCUSSION TITLE
Poller update every time ?

### DISCUSSION THREAD

**Discussion by SawJ83** [thurban/plugin_weathermap#156] [General]
I have quite a lot of maps that are generated over a long period of time.
If I select map generation every puller cycle in the settings, then as I understand it, quite a lot of map generators are launched and take up all the server resources.
Is it possible to check if the map generation has already started?

**Comment by TheWitness** [comment]
Log it as a feature request.  Maybe ask how many concurrent processes.

**Comment by TheWitness** [comment]
I've released 1.3.1 that includes process interlocking.  Give it a test drive.
